pkg/http/core/kubernetes/deploy.go

```diff
@@ -60,6 +68,8 @@ func Deploy(c *gin.Context, dm DeployManifestRequest) {
 
 	manifests := []map[string]interface{}{}
 
+	application := dm.Moniker.App
+
```

Comment:
Remove newlines on 72 and 70. The comment on 73 separates the assignment from the for loop.